docs/deployment.md

```diff
@@ -0,0 +1,34 @@
+# Architecture-Overview
+
+![architecture-overview](images/ci_cd_github_big_picture_public.drawio.png)
+_The picture shows the architecture overview. The steps are explaind in the following passages_
+
+# Source Repo
+
+The top left from the architecture-overview is explaint in [link](./workflows.md).
+
+# Helm Chart
+
+![architekture-external-helm-chart](images/external-helm-chart.drawio.png)
+
+The top right from the architecture-overview is explaint in the following.
+
+We provide a sample chart. https://github.com/it-at-m/helm-charts/tree/main/charts/sps-sample, which consists subcharts. Each subchart is a different module for example frontend, backend, eai. Each module was created with `helm create` and we made some adaptions. The Chart.yml includes the external dependency of the [RefArch Api Gateway Helm Chart](https://github.com/it-at-m/helm-charts/tree/main/charts/refarch-gateway). The main config is done in the values.yml of the parent chart. In the sections refarch-gateway, frontend, backend you can configure the subcharts.
+
+# Deployment
+
+The bottom part from the architecture-overview is expliant in the following.
```

Comment:
würde ich streichen

Reply:
https://github.com/it-at-m/lhm_actions/pull/15#discussion_r1965290406 hier hast du gesagt, es ist nicht klar wo ich etwas erkläre. dann streiche ich es wieder